### PR TITLE
chore: added v2 capabilities for the sdks

### PR DESCRIPTION
**Pull request by jsalaber**

### CODE REVIEW

File: harness/types/capabilities.ts [hunk 1]
```diff
@@ -48,6 +51,7 @@ let sdkCapabilities: { [key: string]: string[] } = {
         Capabilities.cloud,
```
Comment (suthar26) [inline harness/types/capabilities.ts:51]: could you please change this to `local`

Comment (jsalaber) [reply harness/types/capabilities.ts:51]: wait what's wrong with java having cloud capabilities

Comment (jsalaber) [reply harness/types/capabilities.ts:51]: also there is no local option, seems to be the default

Comment (suthar26) [reply harness/types/capabilities.ts:51]: oh wait sorry, not what i meant to comment on

File: harness/types/capabilities.ts [hunk 2]
```diff
@@ -24,18 +24,21 @@ let sdkCapabilities: { [key: string]: string[] } = {
...
+        Capabilities.v2Config,
     ],
     Python: [
         Capabilities.cloud,
         Capabilities.edgeDB,
         Capabilities.clientCustomData,
+        Capabilities.v2Config,
     ],
     DotNet: [
         Capabilities.cloud,
```
Comment (suthar26) [inline harness/types/capabilities.ts:44]: can you add `local` option to DotNet

**Review by suthar26 — APPROVED** [review]

**Pull request merged**